templates/bot/ts/command-and-response/package.json

```diff
@@ -10,7 +10,7 @@
     "dev": "nodemon -e ts --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
     "build": "tsc --build",
     "start": "node ./lib/src/index.js",
-    "test": "npm run build && mocha --require ts-node/register \"tests/**/*.test.ts\"",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
I am not sure if we need to remove the whole test script or keep as this. Let's sync tomorrow with team.